README.md

```diff
@@ -22,18 +18,24 @@ Thus, this program can be utilised to draw paths for automation purposes which w
 
 
 On the other hand, the car will also be able to be controlled via a joystick and save the path that was driven.
-This could be beneficial to, for example, Roombas (cleaning robot) as the user can manually set a path  once and 
+This could be beneficial to, for example, Roombas (cleaning robot) as the user can manually drive a path once and 
 then the roomba can save that path, so it is always able to clean the room/house in that specific order.
 
 ### How are we going to make it?
 
-The mobile app will allow the user to create a grid-based drawing or upload a pre-made image on their Android phone; both of which will be 
-converted into a path that the car can follow.
-In the case of an uploaded image, image-processing will be used to convert the image to movement instructions for the car.
+The mobile app will allow the user to create a grid-based drawing which will be converted into a set of instructions that the car can execute.
+The set of instructions that is pulled from a grid-based drawing will be saved along with the image of the drawing.
 With the use of the sensors and the SmartCar library, the car can follow the path safely and properly.
 
+The joystick will be made from scratch. To make the joystick custom circle drawables, Math library and OnTouch functionality of views will be used.
+The joystick will publish speed and angle to the car via a broker. Arduino codebase will recieve and execute these instructions.
+Saving of instructions will done with GSON by transforming necessary instructions into JSON.
+For the camera, a premade template will be used with along with a connection via a MQTT broker. 
```

Comment:
I think MQTT should be mentioned as a general thing rather than just for the camera (as all parts of the app need MQTT to function)